### PR TITLE
Add colons after agreement labels

### PR DESCRIPTION
**Pull request by lechacon**
null

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-cli/pull/1440)

### CODE REVIEW

**Review by JohnMcPMS — APPROVED** [review]

File: src/AppInstallerCLICore/Workflows/ShowFlow.cpp [hunk 1]
```diff
@@ -97,7 +97,7 @@ namespace AppInstaller::CLI::Workflow
             {
                 if (!agreement.Label.empty())
                 {
-                    info << Execution::ManifestInfoEmphasis << agreement.Label << " ";
+                    info << Execution::ManifestInfoEmphasis << agreement.Label << ": ";
```
Comment (yao-msft) [inline src/AppInstallerCLICore/Workflows/ShowFlow.cpp:100]: If possible, can you add the colons to source agreement labels too? I think I followed the package agreements style when doing source agreements work

Comment (lechacon) [reply src/AppInstallerCLICore/Workflows/ShowFlow.cpp:100]: Added.

File: src/AppInstallerCLICore/Workflows/WorkflowBase.cpp [hunk 2]
```diff
@@ -158,7 +158,7 @@ namespace AppInstaller::CLI::Workflow
             {
                 if (!agreement.Label.empty())
                 {
-                    context.Reporter.Info() << Execution::SourceInfoEmphasis << Utility::LocIndString{ agreement.Label } << " ";
+                    context.Reporter.Info() << Execution::SourceInfoEmphasis << Utility::LocIndString{ agreement.Label } << ": ";
```
Comment (JohnMcPMS) [inline src/AppInstallerCLICore/Workflows/WorkflowBase.cpp:161]: Realized that they should really have `_liv` for some magical future where I get around to enforcing localization.

Comment (lechacon) [reply src/AppInstallerCLICore/Workflows/WorkflowBase.cpp]: Added it to all these strings in Show. I realized there are still some non-localized strings for showing dependencies. I'll update these on my next change.

**Pull request merged**